src/storm-pars/transformer/ParametricTransformer.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+#include "storm/models/sparse/Dtmc.h"
+#include "storm/adapters/RationalFunctionAdapter.h"
+#include "storm/models/sparse/StandardRewardModel.h"
+
+
+namespace storm {
+namespace transformer {
+std::shared_ptr<storm::models::sparse::Dtmc<storm::RationalFunction>> makeRewardsConstant(std::shared_ptr<storm::models::sparse::Dtmc<storm::RationalFunction>> const& pMC);
```

Comment:
We typically do not require a shared pointer as argument if we could also just take the object by reference. Is there a need to require the shared pointer here?